<launch>
  <node name="accel_decel" pkg="accel_decel" type="accel_decel" output="screen" />
  <node name="pid_control" pkg="nemcon" type="pid_control" output="screen">
    <!-- 前：1，右：2，後：3，左：4 -->
    <param name="imu_P" value= "50"/>
    <param name="imu_I" value= "5.5"/>
    <param name="imu_D" value="0.05" />
    <param name="enc_P" value="10.0" />
    <param name="enc_I" value="2.00"/>
    <param name="enc_D" value= "0.02" />
    <param name="v_P" value="210.00" />
    <param name="v_I" value="10.0" />
    <param name="v_D" value="0.009" />
    <param name="vs_P" value="210.0" />
    <param name="vs_I" value="10.0" />
    <param name="vs_D" value="0.009" />
    <param name="lrfv_P" value="77.0" />
    <param name="lrfv_I" value="240.0" />
    <param name="lrfv_D" value="0.005" />
  </node>
</launch>
